integration/system/event_test.go

```diff
@@ -58,3 +65,55 @@ func TestEvents(t *testing.T) {
 	}
 
 }
+
+// Test case for #18888: Events messages have been switched from generic
+// `JSONMessage` to `events.Message` types. The switch does not break the
+// backward compatibility so old `JSONMessage` could still be used.
+// This test verifies that backward compatibility maintains.
+func TestEventsBackwardsCompatible(t *testing.T) {
+	defer setupTest(t)()
+	ctx := context.Background()
+	client := request.NewAPIClient(t)
+
+	since := request.DaemonTime(ctx, t, client, testEnv)
+	ts := strconv.FormatInt(since.Unix(), 10)
+
+	cID := container.Create(t, ctx, client)
+
+	// In case there is no events, the API should have responded immediately (not blocking),
+	// The test here makes sure the response time is less than 3 sec.
+	expectedTime := time.Now().Add(3 * time.Second)
+	emptyResp, emptyBody, err := req.Get("/events")
+	require.NoError(t, err)
+	defer emptyBody.Close()
+	assert.Equal(t, emptyResp.StatusCode, http.StatusOK)
```

Comment:
"expected" and "actual" values are flipped here

Reply:
Thanks @thaJeztah. Done.